cli/step_activator.go

```diff
@@ -40,9 +40,23 @@ func (a stepActivator) activateStep(
 			return "", "", err
 		}
 
+		exist, err := pathutil.IsDirExists(stepAbsLocalPth)
+		if err != nil {
+			return "", "", fmt.Errorf("failed to activate local step: failed to check if a directory exists at %s: %w", stepAbsLocalPth, err)
+		} else if !exist {
+			return "", "", fmt.Errorf("failed to activate local step: the provided directory (%s) doesn't exist", stepAbsLocalPth)
```

Comment:
nit: I think this format a bit more consistent with standard Go errors:

```suggestion
			return "", "", fmt.Errorf("failed to activate local step: the provided directory doesn't exist: %s", stepAbsLocalPth)
```